nrich-search-repository-api/README.md

```diff
@@ -4,5 +4,6 @@
 
 ## Overview
 
-Contains classes that represent the repository API (repository interfaces and factories) for `nrich-search` module. The interfaces have default implementations provided in `nrich-search` module but
-users can provide their own implementations. Unless some overriding is required only classes from this module should be used directly, other configuration should be provided through property files.
+This module contains classes that represent the repository API (repository interfaces and factories) for [nrich-search](../nrich-search/README.md) module.
```

Comment:
Not sure about the `This module` prefix which seems to be used in every README.MD file in this PR.
I think it's evident that documentation is about this module. For example, in this case, I'd prefer `Contains classes that represent...`.

Reply:
I agree. I will change similar occurrences.

Reply:
Based on this [link](https://dictionary.cambridge.org/grammar/british-grammar/ellipsis) it is allowed, but in an informal setting. Because of that, maybe it is better to leave `This module` prefix or at least directly reference the module by writting `It contains...`.

Reply:
We can leave the `This module` prefix in that case.